src/Task/FileSize.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Configuration\GrumPHP;
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class FileSize implements TaskInterface
+{
+    /**
+     * @param GrumPHP $grumPHP
+     */
+    public function __construct(GrumPHP $grumPHP)
+    {
+        $this->grumPHP = $grumPHP;
+    }
+
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return 'file_size';
+    }
+
+    /**
+     * @return array
+     */
+    public function getConfiguration()
+    {
+        $configured = $this->grumPHP->getTaskConfiguration($this->getName());
+
+        return $this->getConfigurableOptions()->resolve($configured);
+    }
+
+    /**
+     * @return OptionsResolver
+     */
+    public function getConfigurableOptions()
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults([
+            'max_size' => '10M',
+        ]);
+
+        $resolver->addAllowedTypes('max_size', ['string']);
```

Comment:
It is also possible to just add a number in bytes without a suffix. (see \Symfony\Component\Finder\Comparator\NumberComparator)
So maybe it should also be possible to allow ints?